problems/strings.js

```diff
@@ -154,8 +227,36 @@ function reverseString() {}
  * example: reverseSentenceWords("i am a racecar") returns "i ma a racecar"
  */
 
-function reverseSentenceWords() {}
+function reverseSentenceWords(str) {
+  let wordsArr = str.split(" ")
+  const reverseWordsArr = []
+  for(let i =0; i < wordsArr.length;i++){
+    let currentWord = wordsArr[i]
+    let reverseWord = reverseString(currentWord)
```

Comment:
Great solution here too! you already solved a portion of this above, so using that function here is an excellent way to solve this one too.